vortex-array/src/array/null/as_arrow.rs

```diff
@@ -0,0 +1,53 @@
+//! Implementation of the [AsArrowArray] trait for [ConstantArray] that is representing
+//! [DType::Null] values.
+
+use std::sync::Arc;
+
+use arrow_array::{ArrayRef as ArrowArrayRef, NullArray as ArrowNullArray};
+use vortex_dtype::DType;
+use vortex_error::{vortex_bail, VortexResult};
+
+use crate::array::null::NullArray;
+use crate::compute::as_arrow::AsArrowArray;
+use crate::{ArrayDType, ArrayTrait};
+
+impl AsArrowArray for NullArray {
+    fn as_arrow(&self) -> VortexResult<ArrowArrayRef> {
+        if self.dtype() != &DType::Null {
```

Comment:
Can probably get rid of this, since it's guaranteed

Reply:
done